MainModule/Client/UI/Default/FPS.lua

```diff
@@ -0,0 +1,48 @@
+client,service = nil,nil
+
+return function(data,env)
+	if env then
+		setfenv(1, env)
+	end
+	
+	local gfps = true
+	local gTable
+	
+	--warn(math.floor(1/game:GetService("RunService").RenderStepped:Wait()))
+	
+	local window = client.UI.Make("Window",{
+		Name  = "FPS";
+		Title = "FPS";
+		Icon = client.MatIcons.Leaderboard;
+		Size  = {150, 70};
+		Position = UDim2.new(0, 10, 1, -80);
+		AllowMultiple = false;
+		NoHide = true;
+		Walls = true;
+		SizeLocked = true;
+		CanKeepAlive = true;
+		OnClose = function()
+			gfps = false
+		end
+	})
+	
+	if window then
+		local label = window:Add("TextLabel",{
+			Text = "...";
+			BackgroundTransparency = 1;
+			TextSize = 20;
+			Size = UDim2.new(1, 0, 1, 0);
+			Position = UDim2.new(0, 0, 0, 0);
+			--TextScaled = true;
+			--TextWrapped = true;
+		})
+
+		gTable = window.gTable
+		window:Ready()
+
+		repeat
+			label.Text = `{math.floor(1/game:GetService("RunService").RenderStepped:Wait())} fps`
```

Comment:
Using the service table is preferred iirc
```suggestion
			label.Text = `{math.floor(1/service["RunService"].RenderStepped:Wait())} fps`
```